src/helper_functions.jl

```diff
@@ -529,15 +529,16 @@ end # @eval
 end # if
 
 """
-    subtypes(interface::Type, concrete::Bool=true)
+    subtypes(interface::Type, concrete::Bool=false)
 
 Return the concrete subtypes of a given interface.
 
 ### Input
 
 - `interface` -- an abstract type, usually a set interface
-- `concrete` -- (optional, default: `true`) if `true`, seek further the inner
-                subtypes of the given interface
+- `concrete`  -- (optional, default: `false`) if `true`, seek further the inner
+                 abstract subtypes of the given interface, otherwise return abstract
+                 subtypes of `interface`
```

Comment:
otherwise return abstract subtypes of `interface`
↓
otherwise return only the direct subtypes of `interface`